integration/mock-backend-server/pom.xml

```diff
@@ -42,6 +42,10 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+        </dependency>
```

Comment:
Aren't we use log4j for logging in integration tests?

Reply:
For some reason the logs for mockbackend does not get printed when I used log4j. Then when changed to a slf4j logger created using LogFactory the logs appear for the container. I just double checked by using a log4j logger again. This is still the case.